reference-implementation/lib/helpers.js

```diff
@@ -62,6 +62,10 @@ exports.IsFiniteNonNegativeNumber = v => {
 };
 
 exports.InvokeOrNoop = (O, P, args) => {
+  assert(exports.typeIsObject(O) === true);
+  assert(typeof P === 'string');
+  assert(Array.isArray(args));
```

Comment:
This is not accurate; the spec says "If args was not passed, let args be a new empty List."